ddl/ddl.go

```diff
@@ -508,8 +508,7 @@ func (d *ddl) asyncNotifyWorker(job *model.Job) {
 	}
 
 	var worker *worker
-	jobTp := job.Type
-	if admin.MayNeedBackfill(jobTp) {
+	if jobNeedBackFill(job, nil, d.store) {
```

Comment:
@zimulala Can we notify both workers regardless of the type of the job?

Reply:
If the notification is incorrect, tikv needs to be read once. It is better to inform correctly if you can, but occasionally there is an error is not a big problem